azure/datalake/store/lib.py

```diff
@@ -238,7 +255,7 @@ def call(self, op, path='', **kwargs):
                              keys - allowed)
         params = {'OP': op}
         params.update(kwargs)
-        func = getattr(requests, method)
+        func = getattr(self.session, method)
```

Comment:
sometimes this is appears to be resulting in a "NoneType" during retry of operations based on the error message:
RuntimeError('Max number of ADL retries exceeded: exception AttributeError(**"\'NoneType\' object has no attribute \'put\'"**,)',)